package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "@babel/preset-react": "^7.0.0",
     "@rails/webpacker": "^4.0.2",
+    "babel-eslint": "^10.1.0",
```

Comment:
As it is used only at build time, I think it will be better if we keep babel-eslint in `devDependencies`